src/OpenTelemetry.Instrumentation.Process/MeterProviderBuilderExtensions.cs

```diff
@@ -49,6 +49,14 @@ public static class MeterProviderBuilderExtensions
 
         builder.AddMeter(ProcessMetrics.MeterName);
 
-        return builder.AddInstrumentation(() => new ProcessMetrics(options));
+        return builder.AddInstrumentation(() =>
+        {
+            if (ProcessMetrics.TryCreate(options, out ProcessMetrics? processMetrics))
+            {
+                return processMetrics;
+            }
+
+            throw new InvalidOperationException("It is not supported to have multiple meterProviders dependency injected ProcessMetrics.");
```

Comment:
Should we really throw exception here?
Please check discussion under https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/935

We should silently continue/adding logs instead of throwing exception.

Reply:
Thanks @Kielek, please checkout this PR for the latest change:
https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/972